model-optimizer/mo/back/ie_ir_ver_2/emitter.py

```diff
@@ -440,9 +440,10 @@ def port_renumber(graph: Graph):
         node = Node(graph, node)
         if node.kind == 'op':
```

Comment:
nit: Let's refactor lines 439-441 with just one line: `for node in graph.get_op_nodes():`

Reply:
Done.